proto/server.proto

```diff
@@ -277,4 +280,45 @@ message StoreCredentialRequest {
 message StoreCredentialResponse {
   StatusCode status = 1;
   string statusMessage = 2;
-}
\ No newline at end of file
+}
+
+message ClusterGitoptsConfig {
+  string usecase = 1;
+  string projectUrl = 2;
+  string accessToken = 3;
+  string  status=4;
+}
+
+message SetClusterGitoptsProjectRequest {
+  string clusterId = 1;
+  string gitoptsUsecase = 2;
+  string projectUrl = 3;
+  string  accessToken=4;
```

Comment:
Need bit change for credential

change accessToken to "credential"

expected to pass json text for "credential"
for ex: 
"credential" : {"accessToken": "token"}